openc3/lib/openc3/microservices/decom_microservice.rb

```diff
@@ -124,13 +124,21 @@ def limits_change_callback(packet, item, old_limits_state, value, log_change)
         when :BLUE, :GREEN, :GREEN_LOW, :GREEN_HIGH
           @logger.info message
         when :YELLOW, :YELLOW_LOW, :YELLOW_HIGH
+          notification = NotificationModel.new(
+            time: time_nsec,
+            severity: "caution",
+            url: "/tools/limitsmonitor",
+            title: "#{packet.target_name} #{packet.packet_name} #{item.name} #{item.limits.state}",
+            body: "Item went into #{item.limits.state} limit status."
+          )
+          NotificationsTopic.write_notification(notification.as_json(:allow_nan => true), scope: @scope)
```

Comment:
This is fairly significant: I'm now adding all yellow limits violations to the notifications. I think this should have been done all along but maybe it should wait until https://github.com/OpenC3/cosmos/issues/694?